### ISSUE TITLE
Markdown support

### ISSUE THREAD

**Issue by cyberbiont** [codesandbox/codesandbox-client#4066]
# 🐛 bug report

When I select a Markdown file in the sandbox., I get the pop-up window with error
"Activating extension 'vscode.markdown-language-features' failed: invalid identity escape in regular expression."

**Comment by garethx** [comment]
Do you have a link to a sandbox where you're seeing this issue?

**Comment by cyberbiont** [comment]
I have digged a little deeper into it and it seems that it happens in Firefox only.
Any sandbox is susceptable, this one I created from Vanilla template and just added README.md.
Select README.md and the error will pop up.
https://codesandbox.io/s/markdown-test-w1kvz?file=/README.md
It only happens first time after you open / reload the sandbox.

In Chrome there's no error, but I'm not able to see Markdown preview, is it not supported?
